File: Modul 2/Soal 4/PRAK204-2310817210019-NaufalElyzar.py
r = float(input())
t = float(input())

# π = 22 / 7

V = 22 * ((r ** 2) / 7) * t
L = (2 * 22 * (r / 7) * t) + (2 * 22 * ((r ** 2) / 7))
K = 2 * 22 * (r / 7)

print(f"Volume bejana adalah {V:.2f}")
print(f"Luas permukaan bejana adalah {L:.2f}")
print(f"Keliling alas bejana adalah {K:.2f}")

File: Modul 2/Soal 4/PRAK204-2310817210019-NaufalElyzar.c
#include <stdio.h>
#include <math.h>

int main() {
    float r, t;

    scanf("%f", &r);
    scanf("%f", &t);

    // π = 22 / 7
    float V = 22 * (pow(r, 2) / 7) * t;

    float L = 2 * 22 * (r / 7) * t + 2 * 22 * (pow(r, 2) / 7);

    float K = 2 * 22 * (r / 7);

    printf("Volume bejana adalah %.2f\n", V);
    printf("Luas permukaan bejana adalah %.2f\n", L);
    printf("Keliling alas bejana adalah %.2f\n", K);

    return 0;
}
